src/storage/src/hummock/state_store.rs

```diff
@@ -269,17 +258,18 @@ impl StateStore for HummockStorage {
         StoreLocalStatistic::flush_all();
     }
 
-    fn clear_shared_buffer(&self) -> impl Future<Output = StorageResult<()>> + Send + '_ {
-        self.min_current_epoch
-            .store(HummockEpoch::MAX, MemOrdering::SeqCst);
-        async move {
-            let (tx, rx) = oneshot::channel();
-            self.hummock_event_sender
-                .send(HummockEvent::Clear(tx))
-                .expect("should send success");
-            rx.await.expect("should wait success");
-            Ok(())
-        }
+    async fn clear_shared_buffer(&self) -> StorageResult<()> {
+        let (tx, rx) = oneshot::channel();
+        self.hummock_event_sender
+            .send(HummockEvent::Clear(tx))
+            .expect("should send success");
+        rx.await.expect("should wait success");
+
+        let epoch = self.pinned_version.load().max_committed_epoch();
+        self.min_current_epoch.store(epoch, MemOrdering::SeqCst);
```

Comment:
After examining the code related `min_current_epoch`, we may remain setting it as `HummockEpoch::MAX`.

Reply:
Sure.